src/core.c/CompUnit/Repository/Perl5.pm6

```diff
@@ -1,51 +1,56 @@
 class CompUnit::Repository::Perl5 does CompUnit::Repository {
-    method need(
-        CompUnit::DependencySpecification $spec,
-        CompUnit::PrecompilationRepository $precomp = self.precomp-repository(),
-        --> CompUnit:D)
-    {
+    my $perl5-dependency :=
+      CompUnit::DependencySpecification.new(:short-name<Inline::Perl5>);
```

Comment:
Good point.  I had thought about that but somehow thought it could be a bad idea (thinking about the IO issue on a resource recently).  Fixed with https://github.com/rakudo/rakudo/commit/29ba6aa3db